examples/data-objects/webflow/src/host/host.ts

```diff
@@ -13,7 +12,8 @@ import { debug } from "./debug";
 import * as styles from "./index.css";
 import { SearchMenuView } from "./searchmenu";
 
-export class WebflowView implements IFluidHTMLView {
+export class WebflowView {
```

Comment:
You'll need to revert these changes and also in package.json for this PR -- converting `WebflowView` to no longer be an `IFluidHTMLView` can be its own PR (probably converting it to React).

Reply:
fixed in d9e3a6d27b08676bc8d282d4e082c45a4b31582c